src/openforms/submissions/tests/test_variables/test_num_queries.py

```diff
@@ -127,8 +127,11 @@ def test_evaluate_form_logic_with_rules(self):
         # 8. Delete submission attachment files
         # 9. RELEASE SAVEPOINT
         # 10. Delete submission values
-        # 11. Create log for rules
-        with self.assertNumQueries(11):
+        # 11. Retrieve formsteps
+        # 12. Retrieve form_defintion related to formstep 1
+        # 13. Retrieve form_defintion related to formstep 2
+        # 14. Creation of timelinelog
+        with self.assertNumQueries(14):
```

Comment:
we'll want to do something about this, because the number of queries is quickly growing and that hurts performance